bika/lims/browser/analysisrequest/reject_samples.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE
+#
+# Copyright 2019 by it's authors.
+# Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
+
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from bika.lims import api
+from bika.lims import bikaMessageFactory as _
+from bika.lims import logger
+from bika.lims import workflow as wf
+from bika.lims.browser import BrowserView, ulocalized_time
+from bika.lims.catalog.analysisrequest_catalog import \
+    CATALOG_ANALYSIS_REQUEST_LISTING
+from bika.lims.utils.analysisrequest import notify_rejection
+from plone.memoize import view
+
+
+class RejectSamplesView(BrowserView):
+    """View that renders the Samples rejection view
+    """
+    template = ViewPageTemplateFile("templates/reject_samples.pt")
+
+    def __init__(self, context, request):
+        super(RejectSamplesView, self).__init__(context, request)
+        self.context = context
+        self.request = request
+        self.back_url = self.context.absolute_url()
+        self.notify = api.get_setup().getNotifyOnSampleRejection()
+
+    def __call__(self):
+        form = self.request.form
+
+        # Form submit toggle
+        form_submitted = form.get("submitted", False)
+
+        # Buttons
+        form_continue = form.get("button_continue", False)
+        form_cancel = form.get("button_cancel", False)
+
+        # Is Rejection Workflow Enabled
+        if not api.get_setup().isRejectionWorkflowEnabled():
```

Comment:
more aesthetically, but you could do a property or method for this as well, maybe:

```
def is_rejection_workflow_enabled(self):
     return api.get_setup().isRejectionWorkflowEnabled()
```